packages/api/endpoints/executions.js

```diff
@@ -44,7 +44,7 @@ function get(event, cb) {
  */
 function handler(event, context) {
   return handle(event, context, true, (cb) => {
-    if (event.httpMethod === 'GET' && event.pathParameters) {
+    if (event.httpMethod === 'GET' && event.pathParameters && event.pathParameters.arn) {
```

Comment:
I have no issue with this per se, but why did it need to be updated?

Reply:
Safety? It's a problem I've run into before so just being safe.

Reply:
It seems like it might be better to handle this in the `get` callback and return an explicit error, as seen in the providers endpoint: https://github.com/nasa/cumulus/blob/master/packages/api/endpoints/providers.js#L31